### PR TITLE
Add flexBasis

### PR DESCRIPTION
**Pull request by browniefed**
Adds the flexBasis property as a responsive style

### CODE REVIEW

**Comment by browniefed** [comment]
I think I got it all, let me know if you need any more changes.

File: README.md [hunk 1]
```diff
@@ -804,6 +807,8 @@ Function Name | Prop       | CSS Property    | Theme Field  | Responsive
 `justifySelf` | `justifySelf` | `justify-self` | none | yes
 `alignSelf` | `alignSelf` | `align-self` | none | yes
 `order` | `order` | `order` | none | yes
+`flexWrap` | `flexWrap` | `flex-wrap` | none | yes
+`flexBasis` | `flexBasis` | `flex-basis` | none | yes
```
Comment (johnotander) [inline README.md]: We should s/none/width/ since we're using the `getWidth` getter.

**Comment by johnotander** [comment]
One little tweak and we're g2g, thx!

**Comment by johnotander** [comment]
Wait, nevermind. I'm being dumb. Ignore my last comment 😬

File: README.md (hunk 1 above)
Comment (browniefed) [reply README.md]: 👍

**Comment by browniefed** [comment]
s/width/none/

**Comment by browniefed** [comment]
Hahha soo width or none?

**Comment by johnotander** [comment]
You had it right the first time, sorry!

**Comment by browniefed** [comment]
Gotcha, it's back to `none` haha. Thankfully github add a squash feature

**Pull request merged**